tests/Avalonia.Markup.Xaml.UnitTests/Xaml/BasicTests.cs

```diff
@@ -772,14 +770,7 @@ public void All_Properties_Are_Set_Before_Final_EndInit()
 <Window xmlns='https://github.com/avaloniaui'
                 xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'
                 xmlns:local='clr-namespace:Avalonia.Markup.Xaml.UnitTests.Xaml;assembly=Avalonia.Markup.Xaml.UnitTests'>
-    <Window.Styles>
-        <Style>
-            <Style.Resources>
-                <x:Double x:Key='Double'>100</x:Double>
-            </Style.Resources>
-        </Style>
-    </Window.Styles>
-    <local:InitializationOrderTracker Width='100' Height='{StyleResource Double}'
+    <local:InitializationOrderTracker Width='100' Height='100'
```

Comment:
The logic of this test has changed. I actually got as far as this in removing `StyleResource` and then found this test failing when I replaced `StyleResource` with `DynamicResource` - did you work out what was happening?

Reply:
Not quite. I'll keep investigating so I can figure it out. It seemed like `Portable.Xaml` didn't like the `x:Double` resource for whatever reason. It was throwing some exception IIRC. I'll take another look.

Reply:
This looks like it's going to need more work than I thought. When the styles are attached, the resources are not propagated to the styled control. @grokys any ideas?

Reply:
I can get the test passing if I call `DelayedBinding.ApplyBindings` on `tracker`, but other than that, the bindings aren't getting initiated.

Reply:
I don't think the original logic of this test actually works with our new resources system since `DynamicResource` always relies on `DelayedBinding` and `StaticResource` relies on `DelayedBinding` to look up resources in styles that have yet to be attached (which is what happens in this case).

Reply:
Ok, I assumed it would be something like this. 👍 with removing this test then.